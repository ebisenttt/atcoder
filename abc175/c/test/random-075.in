-551268955856470 493296496710293 33931516474514
