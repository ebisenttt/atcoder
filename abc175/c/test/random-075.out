25566824210268
